tests/test_metsim.py

```diff
@@ -176,14 +169,12 @@ def test_mtclim(test_setup):
     # Now test the disaggregation as well as forcing generation
     test_setup.params['time_step'] = 60
     test_setup.params['out_vars'] = hourly_out_vars
-    test_setup.load()
-    assert test_setup.ready
 
     # Check to see that the data is valid
     assert type(test_setup.met_data) is xr.Dataset
 
     test_setup.run()
-    hourly = test_setup.output.isel(lat=loc[0], lon=loc[1]).to_dataframe()
+    hourly = test_setup.output.isel(lat=2, lon=2).to_dataframe()
```

Comment:
Had to select one of them.  The corners of this dataset are not in the mask, so it had to be greater than 0, and I was into the number 2 at the time.